app/controllers/matches_controller.rb

```diff
@@ -0,0 +1,37 @@
+class MatchesController < ApplicationController
+
+  def index
+    @id = params[:quiz]
+    if @id == nil or @id == ""
```

Comment:
Eu acho que se usar o @id.blank? ele já faz as duas checagens. Dá uma olhada em alguns métodos do rails para checar elementos nulos/vazios: https://medium.com/le-wagon/how-to-use-nil-blank-present-exists-in-rails-5-fe03e78ab979
```suggestion
    if @id.blank?
```

Reply:
que chique, belezaa